shell.nix

```diff
@@ -1,6 +1,6 @@
 { pkgs ? (import <nixpkgs> {}) }:
 let
-  go = pkgs.go_1_13;
+  go = pkgs.go_1_14;
```

Comment:
Why not go_1_15? (20.09 default)

Reply:
Thanks. I was having toolchain problems in the beginning of this upgrade, so I "downgraded" to 1.14 and put a pin in that, but forgot to come back to it :D It's fixed now without build issues.